### PR TITLE
DP-26179: Implement TF module with a GitHub-to-Teams webhook converter.

### PR DESCRIPTION
**Pull request by DmitriiTartynovLCM**
The `lambda.js` file is a compiled version of the code that includes all the dependencies.

### CODE REVIEW

File: webhook-converters/github-to-teams/README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+Webhook converter: GitHub to MS Teams
+=================
+
+The module defines a lambda function with a publicly accessible URL that accepts .
```
Comment (chrismshelton) [inline webhook-converters/github-to-teams/README.md]: accepts what?

File: webhook-converters/github-to-teams/README.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+For example, it looks for DNS records that point to CloudFront distributions that don't exist anymore.
```
Comment (chrismshelton) [inline webhook-converters/github-to-teams/README.md]: I think this line can be deleted?

**Review by chrismshelton — CHANGES_REQUESTED** [review]
2 Things in the documentation, otherwise looks good to me!

**Comment by DmitriiTartynovLCM** [comment]
@chrismshelton Thanks for catching these issues! Just updated the module description.

**Review by chrismshelton — APPROVED** [review]

**Pull request merged**